back/flag-service/src/user/decorators/spec/UserIdDecorator.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { HttpArgumentsHost } from '@nestjs/common/interfaces';
+import { getUserIdFromCtx } from '../UserId';
+
+describe('UserIdDecorator', () => {
+  describe('getUserIdFromCtx', () => {
+    it ('returns userId', () => {
+      const fakectx = { switchToHttp(): HttpArgumentsHost {
+        return {
+          getRequest<T = any>(): T {
+            return {
+              userId: 'fakeUserId'
+            } as any;
+          }
+        } as any;
+      } } as any;
```

Comment:
It looks like you forgot to address this comment https://github.com/Sorikairox/micronation/pull/7#discussion_r701897217

I'm open to debate on whether my suggestion is good or bad